bin/kondo.sh

```diff
@@ -1,18 +1,16 @@
 #! /usr/bin/env bash
 
-set -euxo pipefail
+# Convenience/CI script for running clj-kondo against all source and test directories using Docker
 
-# Convenience for running clj-kondo against all the appropriate directories.
+# To use a local clj-kondo installation instead of Docker, set KONDO_CMD=clj-kondo
 
-# Copy over Kondo configs from libraries we use.
+set -ex
+
+kondoCmd="${KONDO_CMD:-docker run --rm --volume $PWD:/work --workdir /work cljkondo/clj-kondo:2022.08.03 clj-kondo}"
 
-clj-kondo --copy-configs --dependencies --lint "$(clojure -A:dev -Spath)" --skip-lint --parallel
+# Copy over Kondo configs from libraries we use.
+$kondoCmd --copy-configs --dependencies --lint "$(clojure -A:dev -Spath)" --skip-lint --parallel
```

Comment:
I think this will drop linting drivers if we are using the classpath from `clojure -A:dep -Spath`. We presumably will need `ee:ee-dev:drivers:drivers-dev` as well. I'm also not sure if this will correctly grab the config file that was specified

Reply:
This step is just pulling dependency configs, not linting drivers. That happens in the next command.